src/PolyhedralGeometry/Cone/properties.jl

```diff
@@ -198,7 +198,7 @@ julia> f_vector(square)
 function f_vector(C::Cone)
     pmc = pm_object(C)
     ldim = pmc.LINEALITY_DIM
-    return vcat(fill(0,ldim),pmc.F_VECTOR)
+    return Vector{Int}(vcat(fill(0, ldim), pmc.F_VECTOR))
```

Comment:
```suggestion
    return Vector{fmpz}(vcat(fill(0, ldim), pmc.F_VECTOR))
```
The `F_VECTOR` is a vector of `Polymake.Integer`s so this should be converted to `fmpz`, the numbers in the middle can grow very large.

The `f_vector` for `Polyhedron` could be adjusted as well but I cannot add a suggestion there, and probably `h_vector` and `g_vector` as well, but this could also be done in another PR.